test/python/transpiler/test_unitary_synthesis.py

```diff
@@ -843,19 +844,39 @@ def test_iswap_no_cx_synthesis_succeeds(self):
         result_qc = dag_to_circuit(result_dag)
         self.assertTrue(np.allclose(Operator(result_qc.to_gate()).to_matrix(), cxmat))
 
-    def test_parameterized_basis_gate_in_target(self):
-        """Test synthesis with parameterized RXX gate."""
+    @data(True, False)
+    def test_parameterized_basis_gate_in_target(self, is_random):
+        """Test synthesis with parameterized RZZ gate."""
```

Comment:
Would it make sense to keep both the test for RXX and RZZ? Given that we used to have it...

Reply:
done in [6a82649](https://github.com/Qiskit/qiskit/pull/13568/commits/6a82649709a9ee23f7eef241c055f0f483f00b79)